engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html

```diff
@@ -3,11 +3,6 @@ <h4 data-block="modal-header" translate>Content Host Bulk Content</h4>
 
   <div data-block="modal-body">
     <h4 translate>Assign Lifecycle Environment and Content View</h4>
-    <p>Please run the following commands on the client, so the new Content View can take effect.</p>
-    <pre>
-    # yum clean all
```

Comment:
I'm not sure if `yum clean all` is actually required. Keep in mind that not all content hosts are using yum.

cc @sthirugn

Reply:
@jturel you are right, `yum clean all` is not really required. I also just tested it to make sure.
```
[root@client01 ~]# cat /etc/yum.repos.d/redhat.repo | grep baseurl
baseurl = https://sat-6-7-qa-rhel7.apex.example.com/pulp/repos/Default_Organization/QE/cv1/custom/zooproduct/zoorepo
```
<at this point, I updated the content view from cv1 to cv2 in Satellite UI for this client>

```
[root@client01 ~]# subscription-manager refresh
2 local certificates have been deleted.
All local data refreshed
[root@client01 ~]# cat /etc/yum.repos.d/redhat.repo | grep baseurl
baseurl = https://sat-6-7-qa-rhel7.apex.example.com/pulp/repos/Default_Organization/PROD/cv2/custom/zooproduct/zoorepo
```

Reply:
Removed `yum clean all`